epimodel/exports/epidemics_org.py

```diff
@@ -102,7 +102,7 @@ def extract_smallish_data(
         d = {
             "critical_rates": rates.replace({pd.np.nan: None}).to_dict() if rates is not None else None,
             "hopkins": {
-                "date_index": [x.isoformat() for x in hopkins.index],
+                "date_index": [x.date().isoformat() for x in hopkins.index],
```

Comment:
It worked with latest data  - did it break for you for some reason @mathijshenquet ?

Reply:
It converts the datetime objects into date object so that it does not output `YYYY-MM-DD{time stuff}` but just `YYYY-MM-DD`